hasjob/views/helper.py

```diff
@@ -128,7 +128,7 @@ def load_user_data(user):
                     g.anon_user = anon_user
 
     # Prepare event session if it's not already present
-    if current_auth or g.anon_user and not g.esession:
+    if current_auth and not g.esession:
```

Comment:
because `current_auth` checks for `g.user or g.anon_user`.

Reply:
This is only true if Hasjob is reporting `g.anon_user` to `current_auth` as an actor, which has not been handled yet in this PR. This was reserved for a future PR.